descriptions.go

```diff
@@ -522,6 +696,15 @@ func (ts *TranslatedString) UnmarshalXML(d *xml.Decoder, start xml.StartElement)
 	return nil
 }
 
+func (ts *TranslatedString) validate(errorMsg string) error {
```

Comment:
I abstracted the method so it can be used by validateTranslations() and put a comment there that it therefore cannot be empty anymore. For what I can tell, it's logic to receive a warning when the value is empty there.